premailer/__init__.py

```diff
@@ -1,4 +1,4 @@
 from __future__ import absolute_import, unicode_literals
 from .premailer import Premailer, transform
 
-__version__ = '2.8.1'
+__version__ = '2.8.2'
```

Comment:
Would you mind reverting this one? I don't like to make a new release by merging a PR. It should be done as a separate process.